ng2-components/ng2-activiti-form/src/services/form.service.ts

```diff
@@ -209,6 +214,66 @@ export class FormService {
         return Observable.fromPromise(alfrescoApi.activiti.taskFormsApi.getRestFieldValues(taskId, field));
     }
 
+    /**
+     * Returns a list of child nodes below the specified folder
+     *
+     * @param accountId
+     * @param nodeId
+     * @returns {null}
+     */
+    getAlfrescoNodes(accountId: string, nodeId: string): Observable<[ExternalContent]> {
```

Comment:
I had to use the Activiti service to get the nodes because there is no guarantee that the repository linked to in Activiti is the same one as the demo-shell, also these APIs are needed to create the 'related content' item in Activiti. The code above is now gone anyway and the JS API is used for getting the list of nodes as well as creating the related content.